core/src/test/scala/org/apache/spark/serializer/KryoSerializerSuite.scala

```diff
@@ -76,6 +76,9 @@ class KryoSerializerSuite extends SparkFunSuite with SharedSparkContext {
   }
 
   test("basic types") {
+    val conf = new SparkConf(false)
```

Comment:
You could do this to the next test too, and then, I think you'd have to register Tuple2, but that sounds like a good idea.

And then the next for Scala Map, Seq, List. Again, probably a good idea, as in the JIRA. I actually don't know why this isn't done already.